library/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
     implementation 'org.springframework:spring-aspects'
     //we need to expose it at service module for status checking capability
     api 'org.broadinstitute.dsde.workbench:sam-client_2.13:0.1-6d19a41'
-    implementation 'bio.terra:billing-profile-manager-client:0.1.484-SNAPSHOT'
+    implementation 'bio.terra:billing-profile-manager-client:0.1.611-SNAPSHOT'
```

Comment:
this may not be necessary, but this is the client version of BPM which uses TCL 1.1.38.

Note: I see the Sam client version is way behind. However, changing that requires syntax changes and I don't want to scope-creep this PR.